### PR TITLE
Rename Assessor Portal to Assessor Workbench

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request globally renames "Assessor Portal" to "Assessor Workbench".

## 💭 Motivation and context ##

It is necessary to do this now, since we recently updated Guacamole to use ED25519 SSH keys.  The old Assessor Portal AMI is no longer being created, and hence it cannot support the new SSH key type.  Therefore it is imperative to start using the new Assessor Workbench AMI now.

## 🧪 Testing ##

I used these changes to deploy an Assessor Workbench AMI to our staging COOL environment and verified that it functions as expected.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -533,9 +533,9 @@ the COOL environment.
 | Name | Description |
 |------|-------------|
 | assessment\_private\_zone | The private DNS zone for this assessment. |
-| assessor\_portal\_instance\_profile | The instance profile for the Assessor Portal instances. |
-| assessor\_portal\_instances | The Assessor Portal instances. |
-| assessor\_portal\_security\_group | The security group for the Assessor Portal instances. |
+| assessor\_portal\_instance\_profile | The instance profile for the Assessor Workbench instances. |
+| assessor\_portal\_instances | The Assessor Workbench instances. |
+| assessor\_portal\_security\_group | The security group for the Assessor Workbench instances. |
```
Comment (mcdonnnj) [inline README.md]:
```suggestion
| assessor\_workbench\_instance\_profile | The instance profile for the Assessor Workbench instances. |
| assessor\_workbench\_instances | The Assessor Workbench instances. |
| assessor\_workbench\_security\_group | The security group for the Assessor Workbench instances. |
```

File: assessorworkbench_ec2.tf [hunk 2]
```diff
@@ -56,32 +56,33 @@ resource "aws_instance" "assessorportal" {
...
     aws_security_group.cloudwatch_agent_endpoint_client.id,
-    aws_security_group.assessorportal.id,
+    aws_security_group.assessorworkbench.id,
     aws_security_group.efs_client.id,
     aws_security_group.guacamole_accessible.id,
     aws_security_group.ssm_agent_endpoint_client.id,
   ]
   tags = {
-    Name = format("AssessorPortal%d", count.index)
+    Name = format("Assessorworkbench%d", count.index)
```
Comment (mcdonnnj) [inline assessorworkbench_ec2.tf]:
```suggestion
    Name = format("AssessorWorkbench%d", count.index)
```

File: assessorworkbench_ec2.tf [hunk 3]
```diff
@@ -56,32 +56,33 @@ resource "aws_instance" "assessorportal" {
...
   }
   # volume_tags does not yet inherit the default tags from the
   # provider.  See hashicorp/terraform-provider-aws#19188 for more
   # details.
   volume_tags = merge(data.aws_default_tags.assessment.tags, {
-    Name = format("AssessorPortal%d", count.index)
+    Name = format("Assessorworkbench%d", count.index)
```
Comment (mcdonnnj) [inline assessorworkbench_ec2.tf]:
```suggestion
    Name = format("AssessorWorkbench%d", count.index)
```

File: outputs.tf [hunk 4]
```diff
@@ -4,18 +4,18 @@ output "assessment_private_zone" {
 }
 
 output "assessor_portal_instance_profile" {
```
Comment (mcdonnnj) [inline outputs.tf]:
```suggestion
output "assessor_workbench_instance_profile" {
```

File: outputs.tf [hunk 5]
```diff
@@ -4,18 +4,18 @@ output "assessment_private_zone" {
...
-  value       = aws_iam_instance_profile.assessorportal
-  description = "The instance profile for the Assessor Portal instances."
+  value       = aws_iam_instance_profile.assessorworkbench
+  description = "The instance profile for the Assessor Workbench instances."
 }
 
 output "assessor_portal_instances" {
```
Comment (mcdonnnj) [inline outputs.tf]:
```suggestion
output "assessor_workbench_instances" {
```

File: outputs.tf [hunk 6]
```diff
@@ -4,18 +4,18 @@ output "assessment_private_zone" {
...
-  value       = aws_instance.assessorportal
-  description = "The Assessor Portal instances."
+  value       = aws_instance.assessorworkbench
+  description = "The Assessor Workbench instances."
 }
 
 output "assessor_portal_security_group" {
```
Comment (mcdonnnj) [inline outputs.tf]:
```suggestion
output "assessor_workbench_security_group" {
```

**Review by mcdonnnj — CHANGES_REQUESTED** [review]

**Review by dav3r — APPROVED** [review]
@mcdonnnj beat me to the punch on all the things.  Fix them and I'm 👍

**Review by jasonodoom — APPROVED** [review]

**Review by mcdonnnj — APPROVED** [review]
:shipit:

**Pull request merged**